articles/downloading_data.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Downloading data"
+layout: article
+excerpt: This tutorial will cover the best practices of downloading a file, directory and managing where files are stored
+---
+
+
+## Downloading a file
+
+Every entity in synapse has a unique synapse Id associated with it.  It can be found on every entity page next to `Synapse ID:`, starting with `syn` ending with numbers (ie. `syn00123`)
+
+
+{% tabs %}
+	{% tab Command %}
+		{% highlight bash %}
+synapse get syn00123
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab Python %}
+		{% highlight python %}
+Entity = syn.get("syn00123", downloadFile=False)
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab R %}
+		{% highlight r %}
+Entity = synGet("syn00123", downloadFile= FALSE)
+		{%endhighlight %}
+	{% endtab %}
+
+{% endtabs %}
+
+
+###Downloading File Entities
+`File` entities can be downloaded by using the `get` command. Without specifying the optional parameters, the file entity downloaded will always be the most recent version (See [versioning](http://docs.synapse.org/articles/versioning.html) for more details).
+
+####Versions
+By default, the most recent version of the file is always downloaded unless specified.
+
+{% tabs %}
+	{% tab Command %}
+		{% highlight bash %}
+synapse get syn00123 -v 1
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab Python %}
+		{% highlight python %}
+syn.get("syn00123", version=1)
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab R %}
+		{% highlight r %}
+synGet("syn00123", version=1)
+		{%endhighlight %}
+	{% endtab %}
+
+{% endtabs %}
+
+
+####Download Location
+By default the download location will always be the synapse cache directory, except for the command line client where the default download location is your current working directory.
+
+{% tabs %}
+	{% tab Command %}
+		{% highlight bash %}
+synapse get syn00123 --downloadLocation ./path/to/folder
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab Python %}
+		{% highlight python %}
+syn.get("syn00123", downloadLocation="./path/to/folder")
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab R %}
+		{% highlight r %}
+synGet("syn00123", downloadLocation="./path/to/folder")
+		{%endhighlight %}
+	{% endtab %}
+
+{% endtabs %}
+
+
+####Command line download
```

Comment:
inconsistent Capitalization of Header (doesn't follow others that Capitalize All Words)